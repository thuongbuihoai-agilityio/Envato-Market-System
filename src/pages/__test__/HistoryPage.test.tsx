import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

// Pages
import HistoryPage from '../History';

describe('History Component', () => {
  it('Should render match with snapshot.', async () => {
    const { container } = render(<HistoryPage />);

    expect(container).toMatchSnapshot();
  });
});
