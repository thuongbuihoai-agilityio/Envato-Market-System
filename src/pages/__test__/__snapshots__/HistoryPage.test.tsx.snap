// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`History Component Should render match with snapshot. 1`] = `
<div>
  <div
    class="chakra-stack css-1pkffy6"
  >
    <p
      class="chakra-text css-oz7p8m"
    >
      An error has been occurred!!
    </p>
    <p
      class="chakra-text css-1ly9ogx"
    >
      useLocation() may be used only in the context of a &lt;Router&gt; component.
    </p>
  </div>
</div>
`;
